### DISCUSSION TITLE
MoorDyn-L#N@

### DISCUSSION THREAD

**Discussion by YuYuJou** [OpenFAST/openfast#2534] [Q&A]
I want to output internal force in fairleads.
![image](https://github.com/user-attachments/assets/661a6ebc-439d-4a8d-9552-5c03d82d4ff0)
Are Con5fz & L3N50Fz same?
I output "L3N50pz" and it worked.
But I tried to output "L3N50fz", it said "unsupported".

**Comment by RyanDavies19** [comment]
Hi @YuYuJou, this is a duplicate of #2533. I will answer there
